core/src/datayoga_core/blocks/relational/utils.py

```diff
@@ -39,15 +41,24 @@ def get_engine(connection_name: str, context: Context, autocommit: bool = True)
         lib_dir = connection.get("oracle_thick_mode_lib_dir")
         extra["thick_mode"] = {"lib_dir": lib_dir} if lib_dir else {}
 
+    # PSQL specific parameters to be passed in the query string for SSL connection
+    if db_type == DbType.PSQL:
+        for field in ("sslmode", "sslrootcert", "sslkey", "sslcert", "sslpassword"):
+            if field in connect_args:
+                query_args[field] = connect_args[field]
+                del connect_args[field]
```

Comment:
if you pass those to the query_args, just expect them to be there in the first place.